src/analytics/HookAnalytics.js

```diff
@@ -1,14 +1,14 @@
 import { useEffect, useState, useCallback } from "react";
 import { start } from "./segment";
 
-const HookAnalytics = ({ store }: { store: * }) => {
+const HookAnalytics = ({ store }: { store: * }, user: *) => {
```

Comment:
should'nt this be `{ store, user }: { store: * , user: * }` ?

Reply:
omg you're right